runtime/restart/restart.go

```diff
@@ -128,14 +128,22 @@ func Reconcile(status containerd.Status, labels map[string]string) bool {
 		return false
 	}
```

Comment:
As every switch needs this check, I suggest moving it before the switch to check if there is the `ExplicitlyStoppedLabel` label and return false.